debian/package-postinst

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+set -eu
+
+pg_version=
+installation_message="
++--------------------------------------------------------------+
```

Comment:
nit; I guess we can have 2 extra dashes here as the text width is wider than the seperator lines

Reply:
In the actual output it's not wider, since `$pg_version` is replaced with 10/11/12.